kind.sh

```diff
@@ -24,11 +24,6 @@ trap cleanup EXIT
 export KUBECONFIG="$(kind get kubeconfig-path --name $cluster)"
```

Comment:
This seems to break, investigating.

Reply:
Maybe in https://github.com/kubernetes-sigs/kind/pull/1046. Anyway I think I can do it differently.